data/fields/traffic_signals/sound.json

```diff
@@ -0,0 +1,21 @@
+{
+    "key": "traffic_signals:sound",
+    "type": "combo",
+    "label": "Sound Signals",
+    "strings": {
+        "options": {
+            "yes": "Yes",
+            "no": "No",
+            "locate": {
+                "title": "Locate",
```

Comment:
This matches the raw value, but it feels a bit ungrammatical in this context. [This technology has multiple names in English](http://www.apsguide.org/chapter4_pushbutton.cfm), but “Call Button Locator” would be more grammatical and self-documenting.